### ISSUE TITLE
Image name for multiple images into the same page

### ISSUE THREAD

**Issue by alexbrollo** [danmichaelo/croptool#74]
here: [https://tools.wmflabs.org/croptool/?title=Martini_-_Trattato_di_architettura_civile_e_militare,_1841,_III.djvu&page=27] there's the need to crop two different images from the same page (a pretty frequent case dealing with book illustrations), so there' a need to imagine some "disambiguation" for multiple images (into it.wikisource we add a,b,c.... to the page number)

**Comment by danmichaelo** [comment]
You can already edit the name manually, but it would be easier if CropTool suggested a new name automatically if the file exists. Let's just agree on a naming scheme. Right now the tool only suggests `{file} (page {page} crop).jpg`. If that file exists, we could make it suggest `{file} (page {page} crop 2).jpg` etc… for instance.

Other naming schemes are of course also possible, let me know if you have a preference.

**Comment by alexbrollo** [comment]
You're right, I simply didn't realized that user can edit the file name... :-( 
Tested, perfect.
